QModManager/API/SMLHelper/Patchers/BioReactorPatcher.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace QModManager.API.SMLHelper.Patchers
+{
+    using Harmony;
+    using QModManager.Utility;
+    using System.Collections.Generic;
+
+    internal static class BioReactorPatcher
+    {
+        internal static IDictionary<TechType, float> CustomBioreactorCharges = new SelfCheckingDictionary<TechType, float>("CustomBioreactorCharges", TechTypeExtensions.sTechTypeComparer);
+
+        internal static void Patch(HarmonyInstance harmony)
+        {
+            PatchUtils.PatchDictionary(BaseBioReactor.charge, CustomBioreactorCharges);
```

Comment:
Could we do something to allow patchers to also patch stuff at **run-time**?

Reply:
That would only work for patching of data collections like this.  
For Harmony and TechType patching it wouldn't.

Making the difference apparent to modders using the APIs would be a bit of a challenge, so for the sake of simplicity, it would be easier to just keep them both consistent.